Ginger/Ginger/Drivers/DriversWindows/MobileDriverWindow.xaml

```diff
@@ -73,9 +66,50 @@
                     </StackPanel>
                 </TabItem>
             </TabControl>
+            
+            <TabControl x:Name="xWindowConfigurationsTabs" HorizontalAlignment="Stretch" BorderThickness="1" BorderBrush="{StaticResource $BackgroundColor_White}">
+                <TabItem x:Name="xConfigurationsTab" Style="{StaticResource $CoolTab}" TabIndex="1" >
+                    <TabItem.Header>
+                        <Border Style="{StaticResource $CoolTabBorderStyle}">
+                            <DockPanel >
+                                <UserControls:ImageMakerControl SetAsFontImageWithSize="16" ImageType="Config" Width="20" Height="16" FontWeight="Bold" Foreground="{StaticResource $BackgroundColor_DarkGray}" />
+                                <TextBlock  Text="Configurations" Style="{StaticResource $CoolTabTextBlockStyle}" />
+                            </DockPanel>
+                        </Border>
+                    </TabItem.Header>
+                    <!--Configurations area-->
+                    <ScrollViewer x:Name="xConfigurationsFrame" Grid.Column="3" VerticalScrollBarVisibility="Auto" Background="{StaticResource $BackgroundColor_White}" >
+                        <StackPanel Margin="0,2,0,0" >
+                            <Label Content="Auto refresh mode:" Style="{StaticResource $LabelStyle}" FontWeight="Bold" Margin="0,0,0,0"/>
+                            <StackPanel x:Name="xAutoRefreshModePnl" Orientation="Vertical" Margin="5,0,0,0">
+                                <RadioButton x:Name="xLiveRdBtn" Content="Live" Style="{StaticResource $InputRadioButtonStyle}" GroupName="AutoRefreshMode" VerticalAlignment="Center" IsChecked="True" Checked="xContinualRdBtn_Checked" />
+                                <RadioButton x:Name="xPostOperationRdBtn" Content="Post Operation" Style="{StaticResource $InputRadioButtonStyle}"  GroupName="AutoRefreshMode" VerticalAlignment="Center" Margin="0,0,0,0" Checked="xPostOperationRdBtn_Checked" />
+                                <RadioButton x:Name="xDisabledRdBtn" Content="Disabled" Style="{StaticResource $InputRadioButtonStyle}"  GroupName="AutoRefreshMode" VerticalAlignment="Center" Margin="0,0,0,0" Checked="xDisabledRdBtn_Checked" />
+                            </StackPanel>
+                            <StackPanel x:Name="xRefreshWaitingRatePnl" Orientation="Horizontal" Margin="5,0,0,0">
```

Comment:
is it in seconds? can we try show the unit in tooltip?

Reply:
the configured wait in milisocnds in code is been multiply with the selected number by user so it basically increasing the waiting time in 2-10 times more